docs/index.asciidoc

```diff
@@ -249,6 +232,14 @@ If you'd like to work with the request/response metadata.
 Set this value to the name of the field you'd like to store a nested
 hash of metadata.
 
+[id="plugins-{type}s-{plugin}-password"]
+===== `password` 
+
+  * Value type is <<password,password>>
+  * There is no default value for this setting.
+
+Password to be used in conjunction with the username for HTTP authentication.
```

Comment:
I know you're just reorganizing this bit, but I think we can more clearly link this to the `user` option (instead of generically calling out "the username").

```suggestion
Password to be used in conjunction with <<plugins-{type}s-{plugin}-user>> for HTTP authentication.
```